x-pack/plugins/canvas/public/components/workpad/index.js

```diff
@@ -22,6 +21,7 @@ import {
 import { zoomHandlerCreators } from '../../lib/app_handler_creators';
 import { trackCanvasUiMetric, METRIC_TYPE } from '../../lib/ui_metric';
 import { LAUNCHED_FULLSCREEN, LAUNCHED_FULLSCREEN_AUTOPLAY } from '../../../common/lib/constants';
+import { WorkpadRoutingContext } from '../../routes/workpad';
 import { Workpad as Component } from './workpad';
```

Comment:
nit: it's usually clearer to include the name in front of Component, simply because we know what the component is... usually `Component` is reserved for HOC where you don't necessarily know what you're going to render.  Searching for `<Component` ends up being harder than `<WorkpadComponent`.

```suggestion
import { Workpad as WorkpadComponent } from './workpad';
```